libs/core/src/lib/multi-input/multi-input.component.ts

```diff
@@ -289,6 +294,10 @@ export class MultiInputComponent implements
         if (this.dropdownValues) {
             this.displayedValues = this.dropdownValues;
         }
+        this._subscriptions.add(this._rtlService.rtl.subscribe(isRtl => {
+            this._dir = isRtl ? 'rtl' : 'ltr';
+            this.buildComponentCssClass();
+        }));
```

Comment:
since _rtlService is optional
```suggestion
        if (this._rtlService) {
            this._subscriptions.add(this._rtlService.rtl.subscribe(isRtl => {
                this._dir = isRtl ? 'rtl' : 'ltr';
                this.buildComponentCssClass();
            }));
        }
```

or remove `@Optional()` from `@Optional() private _rtlService: RtlService`